### PR TITLE
Bump to v2.7.1

### PR DESCRIPTION
**Pull request by Onumis**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 * [CHANGE] Bump Rubocop to 0.36 and internal cleanup (preparing for Ruby 2.3)
 
-# 2.6.0 / 2015-01-21
+# 2.6.0 / 2016-01-21
 
 * [FEATURE] Add a minimum score option to the command line interface (by Roberto Schneider)
 * [CHANGE] Display the class and module names when the file has no methods
 
-# 2.5.0 / 2015-01-16
+# 2.5.0 / 2016-01-16
```
Comment (Onumis) [inline CHANGELOG.md]: :see_no_evil:

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+* [BUGFIX] Fix Rubycritic's Analysers running twice on the first run
+
+# 2.7.0 / 2016-01-23
 
 * [FEATURE] Open html report with browser (by YingRui Lu)
 * [CHANGE] Bump Reek to 3.9.1
 * [CHANGE] Bump Rubocop to 0.36 and internal cleanup (preparing for Ruby 2.3)
 
-# 2.6.0 / 2015-01-21
+# 2.6.0 / 2016-01-21
```
Comment (Onumis) [inline CHANGELOG.md]: :see_no_evil:

Comment (troessner) [inline CHANGELOG.md]: Whoopsie!!! :laughing:

**Comment by troessner** [comment]
Ready for merge but travis is red - I just restarted the build so let's see....

**Comment by troessner** [comment]
Seems like travis is having some issues at the moment, restarting itself doesnt work, let's wait till tomorrow.

**Comment by Onumis** [comment]
Yeah, I was about to say that :(

**Comment by Onumis** [comment]
Closing in favor of #120

**Pull request closed without merging**